calculator.js

```diff
@@ -1,5 +1,57 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const prompt = require('prompt');
+
+const calculateUserInput = function (error, promptInput) {
+  calculate(promptInput);
 }
+
+function validNum(input) {
```

Comment:
This is very minor but at Ada (and I think most places) we tend to prefer the variable definition syntax for defining functions rather than this one.
I realize it doesn't help that the Educative course defined them this way :) 
```suggestion
const validNum = function (input) {
```

Reply:
Also with this syntax, no semicolon is needed on line 17 but if you were to switch it to our recommended syntax, a semicolon should be added there.